SAGE Journals.js

```diff
@@ -121,6 +125,11 @@ function scrape(doc, url) {
 				item.abstractNote = abstract;
 			}
 			
+			var tags = ZU.xpathText(doc, '//meta[@name="dc.Subject"]/@content');
```

Comment:
```suggestion
			var tags = doc.querySelectorAll('kwd-group:first-of-type a');
```

Reply:
I guess this would work but thinking about it the example from @zuphilip may not be very representative; it's taken from a [bilingual journal published by editors from French- and English-speaking Canada](https://en.wikipedia.org/wiki/Studies_in_Religion) and I don't think that they intended for users to only use English keywords - shouldn't Zotero reflect all the meta information?

Reply:
It's a judgement call, but I think duplicate tags are pretty useless and the nice thing about this strategy is that the journal appears to put the tags in the language of the article first, so people would get French ones here:
https://journals.sagepub.com/doi/full/10.1177/0008429819858923

Reply:
You're quite right, I did not see that. Let's just hope they continue in the future...